File: Programmers/level1/같은-숫자는-싫어/siwony_.js
function solution(arr)
{
    let answer = [];
    let temp;
    for(let i = 0; i < arr.length; i++){
        if(temp !== arr[i]) {
            temp = arr[i]
            answer[answer.length] = temp;
        }
    }   
    return answer
}

/***테스트 코드***/
console.log(solution([1, 1, 3, 3, 0, 1, 1]))
console.log(solution([4, 4, 4, 3, 3]))

File: Programmers/level1/약수의 합/siwony_.py
def solution(n):
    answer=0
    for i in range(1, n // 2 + 1):   # 약수의 절반이상부터 약수가 없음
        answer += i if n % i == 0 else 0
    return answer + n

#### 테스트 코드 ####
print(solution(12))